src/_data/chipsets.yml

```diff
@@ -9,11 +9,11 @@
   x64: '<span class="material-symbols" style="color: #158477" aria-label="The Flutter SDK supports the x64 architecture on Linux" role="img">verified</span>'
   arm32: '<span class="material-symbols" style="color: #D43324" aria-label="The Flutter SDK does not support the Arm32 architecture on Linux" role="img">dangerous</span>'
   arm64: '<span class="material-symbols" style="color: #158477" aria-label="The Flutter SDK supports the Arm64 architecture on Linux" role="img">verified</span> '
-  risc: '<span class="material-symbols" style="color: #D43324" aria-label="The Flutter SDK does not support the RV64GC architecture on Linux" role="img">dangerous</span>'
+  risc: '<span class="material-symbols" style="color: #DADCE0" aria-label="No version of the Flutter SDK exists for the RV64GC architecture on Linux" role="img">do_not_disturb_on</span>'
```

Comment:
Question: why is this phrased differently for Linux than for macOS and Windows, given all three only support x64 and arm64?

Reply:
All target platforms don't support RV64GC. It is the same language for all platforms ([Windows](https://github.com/flutter/website/pull/10193/files/759a42c219f07baebbfc46a6459730cec4c5f665#diff-275e58b217f338e06bd26b903fb57382eb3d972beef3a73583adcae657b9297cR6), [macOS](https://github.com/flutter/website/pull/10193/files/759a42c219f07baebbfc46a6459730cec4c5f665#diff-275e58b217f338e06bd26b903fb57382eb3d972beef3a73583adcae657b9297cR18), [Linux](https://github.com/flutter/website/pull/10193/files/759a42c219f07baebbfc46a6459730cec4c5f665#diff-275e58b217f338e06bd26b903fb57382eb3d972beef3a73583adcae657b9297cR12), [iOS](https://github.com/flutter/website/pull/10193/files/759a42c219f07baebbfc46a6459730cec4c5f665#diff-275e58b217f338e06bd26b903fb57382eb3d972beef3a73583adcae657b9297cR24), and [Android](https://github.com/flutter/website/pull/10193/files/759a42c219f07baebbfc46a6459730cec4c5f665#diff-275e58b217f338e06bd26b903fb57382eb3d972beef3a73583adcae657b9297cR30)).